### PR TITLE
Adding initial Tomcat and Java setup

### PR DESCRIPTION
**Pull request by amynUP**

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -9,6 +9,8 @@ mod 'puppetlabs-powershell', '2.1.5'
 mod 'puppetlabs-ntp', '7.1.1'
 mod 'puppetlabs-vcsrepo', '2.3.0'
 mod 'puppetlabs-pe_gem', '0.2.0'
+mod 'puppetlabs-tomcat', '2.3.0'
+mod 'puppetlabs-java', '2.4.0'
```
Comment (murdok5) [inline Puppetfile:13]: hey whats java?

File: site/role/manifests/tomcat_role.pp [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class role::tomcat_role {
+
+    include profile::base
```
Comment (murdok5) [inline site/role/manifests/tomcat_role.pp]: Hey we are using a different naming scheme, you can use profile::baseline::linux

**Review by murdok5 — CHANGES_REQUESTED** [review]
See the role comment

**Comment by amynUP** [comment]
@murdok5 Hey, changed baseline. All fixed.

**Review by murdok5 — APPROVED** [review]

**Pull request merged**